### PR TITLE
Updates readme, move env vars directly to test and add test

### PR DESCRIPTION
**Pull request by manuelroth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +86,43 @@ If you work with [Q-cli](https://github.com/nzzdev/Q-cli) there is a config file
...
+
+Install the [Q cli](https://github.com/nzzdev/Q-cli) and start the Q dev server:
+
+```
+$ Q server
+```
+
+Run the Q tool:
+```
+$ npm run index.js
```
Comment (philipkueng) [inline README.md]: `$ node dev.js`

**Review by philipkueng — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (manuelroth) [reply README.md]: Fixed in https://github.com/nzzdev/Q-map/pull/86/commits/339516aa2839816ac1ad5703c7216a16e9294ca9

File: README.md [hunk 2]
```diff
@@ -74,30 +90,105 @@ If you work with [Q-cli](https://github.com/nzzdev/Q-cli) there is a config file
...
+
+#### labelsBelowMap
+
+Allows to show the marker labels below the map.
+
+#### showLegend
+
+This options allows to show the legend. The legend only shows something if features where added as geojson.
+
+#### Display Options
```
Comment (philipkueng) [inline README.md:174]: rename `displayOptions`

Comment (manuelroth) [reply README.md:174]: We decided to leave this, because its not a property but a title

**Review by benib — APPROVED** [review]

**Pull request merged**